app/actions.js

```diff
@@ -407,29 +405,73 @@ const initializeCabal = ({ addr, username, settings }) => async dispatch => {
   const isNew = !addr
   const cabalDetails = isNew ? await client.createCabal() : await client.addCabal(addr)
   addr = cabalDetails.key
-  client.focusCabal(addr)
 
-  let firstUpdate = true
-  cabalDetails.on('update', throttle((details) => {
-    const users = details.getUsers()
-    const username = details.getLocalName()
-    const channels = details.getChannels()
-    const channelsJoined = details.getJoinedChannels()
-    const channelMessagesUnread = getCabalUnreadMessagesCount(details)
-    const currentChannel = details.getCurrentChannel()
-    const channelMembers = details.getChannelMembers()
+  function onUpdate () {
+    const users = cabalDetails.getUsers()
+    const username = cabalDetails.getLocalName()
+    const channels = cabalDetails.getChannels()
+    const channelsJoined = cabalDetails.getJoinedChannels()
+    const channelMessagesUnread = getCabalUnreadMessagesCount(cabalDetails)
+    const currentChannel = cabalDetails.getCurrentChannel()
+    const channelMembers = cabalDetails.getChannelMembers()
     dispatch({ type: 'UPDATE_CABAL', addr, channelMessagesUnread, users, username, channels, channelsJoined, currentChannel, channelMembers })
     dispatch(getMessages({ addr, amount: 1000, channel: currentChannel }))
     dispatch(updateAllsChannelsUnreadCount({ addr, channelMessagesUnread }))
-    if (firstUpdate) {
-      firstUpdate = false
-      // Focus default or last channel viewed
-      dispatch(viewCabal({ addr, channel: settings.currentChannel }))
```

Comment:
i think this should be equivalent to `on("init")` https://github.com/cabal-club/cabal-client/blob/master/api.md#CabalDetails+event_init